src/main/java/org/spongepowered/common/mixin/core/entity/player/EntityPlayerMixin.java

```diff
@@ -233,8 +233,8 @@ private void onAddExperienceLevels(final int levels, final CallbackInfo ci) {
             return;
         }
         if (!this.dontRecalculateExperience) {
-            final int newLevel = Math.max(this.experienceLevel + levels, 0);
-            postEventAndUpdateExperience(ExperienceHolderUtils.xpAtLevel(newLevel)
+            final int newLevel = this.experienceLevel + levels;
+            postEventAndUpdateExperience(newLevel <= 0 ? 0 : ExperienceHolderUtils.xpAtLevel(newLevel)
```

Comment:
Shouldn't this just be `<`, not `<=`? That would be consistent with https://github.com/SpongePowered/Sponge/issues/3346 then.

Reply:
Yes, good catch.